pkg/server/router/did.go

```diff
@@ -152,3 +152,39 @@ func (dr DIDRouter) GetDIDByMethod(ctx context.Context, w http.ResponseWriter, _
 	resp := GetDIDByMethodResponse{DID: gotDID.DID}
 	return framework.Respond(ctx, w, resp, http.StatusOK)
 }
+
+type GetDIDsByMethodResponse struct {
+	DIDs []didsdk.DIDDocument `json:"dids,omitempty"`
+}
+
+// GetDIDsByMethod godoc
+// @Summary      Get DIDs
+// @Description  Get DIDs by method
+// @Tags         DecentralizedIdentityAPI
+// @Accept       json
+// @Produce      json
+// @Param        method   path      string                    true  "Method"
+// @Success      200      {object}  GetDIDsByMethodResponse
+// @Failure      400      {string}  string  "Bad request"
+// @Router       /v1/dids/{method} [get]
+func (dr DIDRouter) GetDIDsByMethod(ctx context.Context, w http.ResponseWriter, _ *http.Request) error {
```

Comment:
This is good chance, can you run mage spec ?

Reply:
ah read this after I merged